supplier_calendar/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 ForgeFlow
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Supplier Calendar",
+    "summary": "Supplier Calendar",
+    'version': '12.0.1.0.0',
+    "category": "Purchase Management",
+    "author": "ForgeFlow,"
+               "Odoo Community Association (OCA)",
+    "maintainers": [
+        'Nuria Martin <nuria.martin@forgeflow.com>',
+        'Jordi Ballester <jordi.ballester@forgeflow.com>',
+        'Lois Rilo <lois.rilo@forgeflow.com>'
+    ],
```

Comment:
mainteiners key is used by ocabot to allow the **github users** specified to merge even if they are not PSC. See this link for more reference: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/oca_module_lifecycle_maintainer_role.rst#maintainer-role-and-responsibilities

So, in short, this should be something like `"maintainers": ["NuriaMForgeFlow"]`